yt_dlp/extractor/servus.py

```diff
@@ -60,85 +84,54 @@ class ServusIE(InfoExtractor):
-            if not format_url:
-                continue
-            extension = resource.get('extension')
-            type_ = resource.get('type')
-            if extension == 'jpg' or type_ == 'reference_keyframe':
-                thumbnail = format_url
-                continue
-            ext = determine_ext(format_url)
-            if type_ == 'dash' or ext == 'mpd':
-                formats.extend(self._extract_mpd_formats(
-                    format_url, video_id, mpd_id='dash', fatal=False))
-            elif type_ == 'hls' or ext == 'm3u8':
-                formats.extend(self._extract_m3u8_formats(
-                    format_url, video_id, 'mp4', entry_protocol='m3u8_native',
-                    m3u8_id='hls', fatal=False))
-            elif extension == 'mp4' or ext == 'mp4':
-                formats.append({
-                    'url': format_url,
-                    'format_id': type_,
-                    'width': int_or_none(resource.get('width')),
-                    'height': int_or_none(resource.get('height')),
-                })
+            'https://api-player.redbull.com/stv/servus-tv?timeZone=Europe/Berlin',
+            video_id, 'Downloading video JSON', query={'videoId': video_id})
+        if 'videoUrl' not in video:
+            self._report_errors(video)
+        formats, subtitles = self._extract_m3u8_formats_and_subtitles(
+            video['videoUrl'], video_id, 'mp4', m3u8_id='hls')
 
-        attrs = {}
-        for attribute in video['attributes']:
-            if not isinstance(attribute, dict):
-                continue
-            key = attribute.get('fieldKey')
-            value = attribute.get('fieldValue')
-            if not key or not value:
-                continue
-            attrs[key] = value
-
-        title = attrs.get('title_stv') or video_id
-        alt_title = attrs.get('title')
-        description = attrs.get('long_description') or attrs.get('short_description')
-        series = attrs.get('label')
-        season = attrs.get('season')
-        episode = attrs.get('chapter')
-        duration = float_or_none(attrs.get('duration'), scale=1000)
+        season = video.get('season')
         season_number = int_or_none(self._search_regex(
             r'Season (\d+)', season or '', 'season number', default=None))
+        episode = video.get('chapter')
         episode_number = int_or_none(self._search_regex(
             r'Episode (\d+)', episode or '', 'episode number', default=None))
 
         return {
             'id': video_id,
-            'title': title,
-            'alt_title': alt_title,
-            'description': description,
-            'thumbnail': thumbnail,
-            'duration': duration,
-            'timestamp': unified_timestamp(video.get('lastPublished')),
-            'series': series,
+            'title': video.get('title'),
+            'description': self._get_description(video_id) or video.get('description'),
```

Comment:
Since _get_description makes an additional API call, wouldn't it be better to do 
```suggestion
            'description': video.get('description') or self._get_description(video_id),
```
or is there some issue with it?

Reply:
No, see further up (https://github.com/yt-dlp/yt-dlp/pull/6036#discussion_r1070457652). `video.description` is basically the title + subtitle only.